src/config/params.yaml

```diff
@@ -17,10 +17,10 @@
 loop_time: 1e-07
 
 # Boolean: Print Data to console while running
-print_data: False
+print_data: True
```

Comment:
Minor nitpick but is this needless overhead since we're already logging?

Reply:
Changed back to False